warehouse/templates/email/admin-reset-2fa/body.html

```diff
@@ -0,0 +1,27 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "email/_base/body.html" %}
+
+{% block content %}
+
+<p>An account recovery request was filed at <a href="{{ gh_link }}">{{ gh_link }}</a> for the PyPI username "{{username}}".</p>
+
+<p>In order to verify the request, please push a branch with the name "{{token}}" to the public source code repository for the {{project_name}} project.</p>
```

Comment:
This doesn't seem to handle the case where `project_name` is `None`.

Reply:
Yup. I haven't really gone through the email templates fully.